build.gradle

```diff
@@ -174,12 +171,12 @@ dependencies {
     compile 'org.jsoup:jsoup:1.12.1'
     compile 'com.mashape.unirest:unirest-java:1.4.9'
 
-    // >1.8.0-beta is required for java 9 compatibility
     compile 'org.slf4j:slf4j-api:2.0.0-alpha0'
-    compile 'org.apache.logging.log4j:log4j-slf4j18-impl:2.12.1'
-    compile 'org.apache.logging.log4j:log4j-jcl:2.12.1'
-    compile 'org.apache.logging.log4j:log4j-api:2.12.1'
-    compile 'org.apache.logging.log4j:log4j-core:2.12.1'
+    compile 'org.apache.logging.log4j:log4j-jcl:3.0.0-SNAPSHOT'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-slf4j18-impl', version: '3.0.0-SNAPSHOT'
```

Comment:
please change this back to the more compact format or we will all get hell of merge conflicts

Reply:
I thought that this is the "new" format, as many other dependencies are specified in this form. Personally, I have no strong opinion here but would prefer the extended form if only for the reason that you can simply copy & paste from maven.

Reply:
It's the same, just more verbose.
We should however be consistent.
I always use https://sagioto.github.io/maven2gradle/ to get the compact variant